packages/legend-application-studio/src/components/editor/side-bar/SideBar.tsx

```diff
@@ -44,8 +44,14 @@ export const SideBar = observer(() => {
         return <WorkspaceUpdater />;
       case ACTIVITY_MODE.CONFLICT_RESOLUTION:
         return <WorkspaceUpdateConflictResolver />;
-      case ACTIVITY_MODE.PROJECT_OVERVIEW:
-        return <ProjectOverview />;
+      case ACTIVITY_MODE.PROJECT_OVERVIEW: {
+        editorStore.applicationStore.navigator.visitAddress(
```

Comment:
So let's take away `ACTIVITY_MODE.PROJECT_OVERVIEW`, remember this is nolonger an activity mode that will show up in the sidebar, it's an action. This is related to https://github.com/finos/legend-studio/issues/1518 - ideally we should do the first point over there first.